routes/routeStack.js

```diff
@@ -1,14 +1,31 @@
 import { createStackNavigator } from "react-navigation-stack";
```

Comment:
How does navigation work now?

Reply:
It was a question of understanding what the navigation variable works. That variable is available to all the screens that we subscribe at this level (RouteStack.js). The first screen on the list is always the first screen shown. I think before it wasn't working because we didn't have a clear understanding of the stackNavigator. But it's working properly now. StackNavigator also takes care of the animation between the screens (customizable as well) and allows us to go back on the previous screen easily.

Reply:
The main screen has to be the first one in the stack. All the other screens are added to the route stack. Navigation between screens is defined in the corresponding component/screens. For example, navigation from the BottomMenu screen to the DoubleSearch screen is defined as follows: 

`const goToDoubleSearchBar = () => {
        navigation.navigate("DoubleSearch", { destinationName: destination });
    };`

As you can see, the props that we need to pass from one screen to another can be included :)